client/src/app/employee/employee-list/state/employee-list.service.ts

```diff
@@ -27,7 +24,7 @@ export class EmployeeListService {
   getEmployees() {
     this.store.setLoading(true);
     return this.http
```

Comment:
I would remove the `return` here. One thing you can do to prevent yourself from accidentally exposing RxJs stuff from a service is to set the return type on these methods to `void`.

Reply:
fixed! good recommendation on adding a return type ! 👍